pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1994,3 +1995,15 @@ func needsMorePCIEControlers(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+func DomainInterfaceSpec(ctx *ConverterContext, attachmentName string, tapDeviceName string) api.Interface {
+	return api.Interface{
+		Type: "ethernet",
+		Target: &api.InterfaceTarget{
+			Device:  tapDeviceName,
+			Managed: "no",
+		},
+		Model: &api.Model{Type: translateModel(ctx, "virtio")},
+		Alias: api.NewUserDefinedAlias(HotplugIfaceNamePrefix + attachmentName),
```

Comment:
Why is the hotplug prefix necessary, can we manage only with Libevirt standard prefix?